cmd/clusterctl/client/cluster/proxy.go

```diff
@@ -171,6 +176,30 @@ func (k *proxy) ListResources(labels map[string]string, namespaces ...string) ([
 		return nil, errors.Wrap(err, "failed to list api resources")
 	}
 
+	// Exclude from discovery the objects from the cert-manager/provider's CRDs.
```

Comment:
nit: Please, update func/interface comments as well to reflect this changes

Reply:
done

Reply:
@Rozzii  I think the idea was to also adjust the godoc in the Proxy interface for the ListResources func (cmd/clusterctl/client/cluster/client.go)